node/core/approval-voting/src/lib.rs

```diff
@@ -2306,6 +2373,8 @@ async fn launch_approval<Context>(
 
 	let candidate_hash = candidate.hash();
 	let para_id = candidate.descriptor.para_id;
+	launched_work_span.add_string_tag("candidate-hash", format!("{:?}", &candidate_hash));
```

Comment:
Why not use `.with_candidate(candidate_hash)` ?

Reply:
Yep, good idea 👍🏼 ⚡